src/codeu/chat/client/commandline/Chat.java

```diff
@@ -278,6 +296,20 @@ public void invoke(List<String> args) {
       }
     });
 
+    panel.register("ci-add", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        final String name = args.size() > 0 ? args.get(0) : "";
+        if (name.length() > 0) {
+          if (user.begin(name) == null) {
```

Comment:
This would actually need to cal a different method entirely, but lest's focus on getting UserInterest to work.